custom-ports/kaldi/portfile.cmake

```diff
@@ -18,4 +18,6 @@ vcpkg_cmake_configure(
    -DKALDI_VERSION=${KALDI_VERSION}
 )
 vcpkg_build_cmake()
-vcpkg_install_cmake()
\ No newline at end of file
+vcpkg_install_cmake()
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
```

Comment:
I noticed some warnings:

```
/Users/runner/work/1/s/custom-ports/kaldi/portfile.cmake: warning: This port creates ${CURRENT_PACKAGES_DIR}/lib/cmake and/or ${CURRENT_PACKAGES_DIR}/debug/lib/cmake, which should be merged and moved to ${CURRENT_PACKAGES_DIR}/share/${PORT}/cmake. Please use the helper function vcpkg_cmake_config_fixup() from the port vcpkg-cmake-config. To suppress this message, add set(VCPKG_POLICY_SKIP_LIB_CMAKE_MERGE_CHECK enabled)
/Users/runner/work/1/s/custom-ports/kaldi/portfile.cmake: warning: the license is not installed to ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright . This can be fixed by adding a call to vcpkg_install_copyright. To suppress this message, add set(VCPKG_POLICY_SKIP_COPYRIGHT_CHECK enabled)
/Users/runner/work/1/s/custom-ports/kaldi/portfile.cmake: note: Consider adding: vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
/Users/runner/work/1/s/custom-ports/kaldi/portfile.cmake: warning: There should be no absolute paths, such as the following, in an installed package. To suppress this message, add set(VCPKG_POLICY_SKIP_ABSOLUTE_PATHS_CHECK enabled)
```

I think these will address these: 

```
vcpkg_cmake_config_fixup()
vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
```

Reply:
How important are these? I think they're only important if we want to upstream the ports, which I don't think we want to.

Putting either one of the in results in an error that I'm not sure how to fix yet.

I can suppress them:

```
set(VCPKG_POLICY_SKIP_LIB_CMAKE_MERGE_CHECK enabled)
set(VCPKG_POLICY_SKIP_ABSOLUTE_PATHS_CHECK enabled)
```

Reply:
Oops I was building Debug and not Release.

`vcpkg_cmake_config_fixup()` doesn't seem to fix anything. I still get warnings if I include that.

`vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")` does seem to fix a warning, so I'll keep that in.

Reply:
Sorry, lots of various messages on portfile.cmake.  I think this would resolve all the various warnings: 8db13b941b8c7ca5bc4077ad225599584e482f15.  I'm testing it out, here: https://dev.azure.com/techsmith/ThirdParty-Packages-vcpkg/_build/results?buildId=402027&view=results, so probably standby for a min until that finishes up before picking that up.

> How important are these? I think they're only important if we want to upstream the ports, which I don't think we want to.
Well, if it's not too difficult, I personally like to address these by fixing or suppressing them if it's not too difficult, just for a lot of the same reasons we convert a lot of our C++ warnings to errors and address those.  

If we want to upstream these, it might be nice to be able to have community support in keeping it up to date, and it's a way to give back to to the vcpkg project, which we are definitely benefitting from.

Reply:
I'm fine with eventually wanting to upstream them, I just don't want that to get in the way of actually getting what we want done. If it's a quick fix, let's just put it in. But I'm pretty certain there's going to be a lot of work to actually getting it to be accepted, as at least one person has tried to upstream a Dawn port and failed.

Reply:
This commit https://github.com/TechSmith/ThirdParty-Packages-vcpkg/commit/8db13b941b8c7ca5bc4077ad225599584e482f15 should be good to pick, if you want to make those changes.  It should resolve all the rest of the warnings on Mac and Windows, so far as I can see ([build #402027](https://dev.azure.com/techsmith/ThirdParty-Packages-vcpkg/_build/results?buildId=402027&view=results)).

Reply:
Thanks! I just cherry-picked it in.